### DISCUSSION TITLE
object fields order in constom plugin schema

### DISCUSSION THREAD

**Discussion by doukai** [dotansimha/graphql-code-generator#8823] [Q&A]
This is my User object define:
`
type User {
    login: String
    passworld: String
    name: String
    age: Integer
}`

but the fields order in plugin function schema parmater is:
`type User {
    age: Integer
    login: String
    name: String
    passworld: String
}`

The order of fields  has changed (alphabetically),
Is there any way to keep the original order?
thanks~

**Comment by saihaj** [comment]
what is the use case for this?

**Reply by Neluji** [reply]
I gave mine [down here](https://github.com/dotansimha/graphql-code-generator/discussions/8823#discussioncomment-9591877) ^^

**Comment by Neluji** [comment]
Hi there, I'm having a similar issue with enums: I use generated enums to derive lists of options, but as the schema order is not respected the lists order are messed up (e.g. I end up having a _Other_ option in the middle of the list)
